.chain-interactions/accounts/query-accounts.md

```diff
@@ -1 +1,667 @@
+        <span data-ty>Providers: 1</span>
+        <span data-ty>Sufficients: 0</span>
+        <span data-ty></span>
+        <span data-ty>Balance Details:</span>
+        <span data-ty>================</span>
+        <span data-ty>Free Balance: 59781317040 (5.978131704 PAS)</span>
+        <span data-ty>Reserved Balance: 0 (0 PAS)</span>
+        <span data-ty>Frozen Balance: 0 (0 PAS)</span>
+        <span data-ty></span>
+        <span data-ty>Total Balance: 59781317040 (5.978131704 PAS)</span>
+        <span data-ty></span>
+        <span data-ty>Disconnected</span>
+    </div>
+
+=== "Polkadot.js"
+
+    !!! warning "Maintenance Mode Only"
+        The Polkadot.js API is no longer actively developed. New projects should use [PAPI](/reference/tools/papi/){target=\_blank} or [Dedot](/reference/tools/dedot/){target=\_blank} as actively maintained alternatives.
+
+    [Polkadot.js API](/reference/tools/polkadot-js-api/){target=\_blank} is a comprehensive JavaScript library with extensive ecosystem support.
+
+    **Prerequisites**
+
+    - [Node.js](https://nodejs.org/){target=\_blank} v18 or higher
+    - npm, pnpm, or yarn package manager
+
+    **Environment Setup**
+
+    1. Create and initialize a new project:
+
+        ```bash
+        mkdir pjs-query-account-example && cd pjs-query-account-example && \
+        npm init -y && npm pkg set type=module
+        ```
+
+    2. Install dependencies:
+
+        ```bash
+        npm install @polkadot/api
+        ```
+
+    **Query Account Data**
+
+    The following example queries account information including balance, nonce, and other account data.
+
+    Create a file named `query-account.js`:
+
+    ```javascript title="query-account.js"
+    import { ApiPromise, WsProvider } from "@polkadot/api";
+
+    const POLKADOT_HUB_RPC = "INSERT_WS_ENDPOINT";
+    const ACCOUNT_ADDRESS = "INSERT_ACCOUNT_ADDRESS";
+
+    async function main() {
+      // Create a WebSocket provider
+      const wsProvider = new WsProvider(POLKADOT_HUB_RPC);
+
+      // Initialize the API
+      const api = await ApiPromise.create({ provider: wsProvider });
+      console.log("Connected to Polkadot Hub");
+
+      console.log(`\nQuerying account: ${ACCOUNT_ADDRESS}\n`);
+
+      // Query account information
+      const accountInfo = await api.query.system.account(ACCOUNT_ADDRESS);
+
+      // Display account information
+      console.log("Account Information:");
+      console.log("===================");
+      console.log(`Nonce: ${accountInfo.nonce.toString()}`);
+      console.log(`Consumers: ${accountInfo.consumers.toString()}`);
+      console.log(`Providers: ${accountInfo.providers.toString()}`);
+      console.log(`Sufficients: ${accountInfo.sufficients.toString()}`);
+      
+      console.log("\nBalance Details:");
+      console.log("================");
+      console.log(`Free Balance: ${accountInfo.data.free.toString()} (${Number(accountInfo.data.free.toBigInt()) / 10_000_000_000} PAS)`);
```

Comment:
Not sure if this is how I would structure this -- the division by `10_000_000_000`. I think I did this differently in another tutorial.